### DISCUSSION TITLE
Building tailwindcss with Postcss on Mac?

### DISCUSSION THREAD

**Discussion by Destaq** [tailwindlabs/tailwindcss#2844] [Help] [answered]
New to tailwindcss here, using version 2.0.1 and a MacOS Catalina on 10.15. A friend on Windows can build with postcss well, but I am unable to do so.

Could anyone help me build tailwindcss with postcss?

Node version: 14.15.1

Does not work for the postcss v7 community build either.

Steps:
1. `npm init --y`
2. `npm install tailwindcss postcss autoprefixer`
3. `npx tailwindcss init`
4.  Create CSS folder with styles.css in it and the @tailwind rules.
5.  Run `npx tailwindcss build ./css/styles.css -o ./css/dist.css`

Sadly, I get the following failure: `true is not a Postcss plugin`

![image](https://user-images.githubusercontent.com/61620873/99828204-eee57c00-2b5a-11eb-80bd-728ef48abe87.png)

```
 "dependencies": {
    "autoprefixer": "^10.0.2",
    "postcss": "^8.1.8",
    "tailwindcss": "^2.0.1"
  }
```

**Comment by adamwathan** [comment]
Super weird! Can you create a project on GitHub that reproduces and share the link?

**Reply by Destaq** [reply]
I ended up asking for help on the Discord server. I'll post the solution here.

**Comment by Destaq** [comment, marked as answer]
`    "build:css": "node ./node_modules/.bin/tailwindcss build ./css/styles.css -o ./css/dist.css"`
